rollup.config.js

```diff
@@ -6,7 +6,8 @@ import { terser } from 'rollup-plugin-terser'
 
 import pkg from './package.json'
 
-const noDeclarationFiles = { compilerOptions: { declaration: false } };
+const extensions = ['.ts', '.js']
```

Comment:
We actually have no .js files anymore. Let's keep it to just .ts.

Reply:
ok, changed it to just `.ts`